Allow to configure access/error log files

bbinet opened the pull request.

bbinet committed:
Allow to configure access/error log files

fpytloun commented on nginx/files/_log.conf:
```diff
@@ -1,3 +1,5 @@
 
-  access_log /var/log/nginx/{{ site_name }}-access.log;
-  error_log /var/log/nginx/{{ site_name }}-error.log;
+  access_log {{ site.get('access_log', server.log_dir + '/' + site_name + '-access.log') }};
```
I would prefer this structure of metadata:

```
site:
  log:
    access: 
    error: 
```

bbinet replied:
Ok, I can change that.

And for the main server logs, should I use the same metadata structure?

fpytloun replied:
Yes, that would be great

bbinet replied:
The structure of log metadata is now updated with commit: d63beda

bbinet committed:
Update the structure of metadata for error/access logs

This now looks like:

```
site:
  log:
    access: /path/to/access.log
    error: /path/to/error.log
```

fpytloun commented:
Thank you
